macros/alter_table_properties.sql

```diff
@@ -775,6 +775,14 @@ ALTER VIEW cow_protocol_ethereum.solvers SET TBLPROPERTIES ('dune.public'='true'
                                                 'dune.data_explorer.contributors'='["bh2smith", "gentrexha"]');
 {% endset %}
 
+{% set cow_protocol_ethereum_trades %}
+ALTER VIEW cow_protocol_ethereum.trades SET TBLPROPERTIES ('dune.public'='true',
+    'dune.data_explorer.blockchains'='["ethereum"]',
+    'dune.data_explorer.category'='abstraction',
+    'dune.data_explorer.abstraction.type'='project',
+    'dune.data_explorer.abstraction.name'='cow_protocol',
+    'dune.data_explorer.contributors'='["bh2smith", "gentrexha"]');
+{% endset %}
```

Comment:
It seems to me that this could go into the block above so that all cow_protocol table modifications are made in the same place (instead of defining a new block for each...)